src/components/Button/Button.component.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import './Button.styles.scss';
+
+function Button(props) {
+  return (
+    <button
+      className={'button ' + props.className}
```

Comment:
You can use [`classnames` library ](https://www.npmjs.com/package/classnames) to concatenate two classes.

```js
import cx from 'classnames';
...
  <button
      className={cx('button', props.className)}
...

```